### ISSUE TITLE
[docs] Add more detail to unsupported platforms

### ISSUE THREAD

**Issue by jamiebuilds** [tauri-apps/plugins-workspace#1867]
In `[package.metadata.platforms]` there are a number of things that are marked `"none"`. For example in `tauri-plugin-sql` for `ios`:

https://github.com/tauri-apps/plugins-workspace/blob/e8069af80401cfc1e90297b786bef05a929ed9a6/plugins/sql/Cargo.toml#L17-L22

I just went the Tauri Discord server to go ask for more info on why it's not supported. (Is it planned? Is there something blocking support? etc)

It would be nice (at least on the docs site) if unsupported platforms were a bit more broken down, either with more `"level"`'s or just more notes being added.

I think this list would be exhaustive:

- **Inapplicable:** Does not apply to this platform (ex: `tauri-plugin-cli` on iOS/Android)
- **Unplanned:** Have not yet seen enough interest in supporting, or just not high enough priority to have been looked into, but it hasn't been rejected
- **Planned:** Maintainers want to support this platform, but work hasn't started.
  - **_For Note:_** If it's on the roadmap for a specific future release
- **Blocked:** Blocked on missing platform APIs or third-party deps don't support the platform.
  - **_For Note:_** Details on what is blocking support for the platform.
- **Wontfix:** Maintainers do not want to support the plugin on this platform (ex: Maintenance cost)
- These two are more workflow things that might be weird to put in the `Cargo.toml` but useful to know:
  - **In Development:** Some work has been done but it's not enabled yet
  - **In Preview:** Available in pre-release version, but not yet in a stable release

**Comment by nprbst** [comment]
I second this request. @jamiebuilds what did you learn about the reason for `tauri-plugin-sql` being unavailable on `iOS`?

**Comment by riungemaina** [comment]
Wondering the same about the `tauri-plugin-sql` I'm lost on where I can get details on why it's not supported.

**Comment by ryusei-48** [comment]
I too wonder why only IOS is not supported.
My current project requires cross-platform support in the future, but I am looking for a way to break the deadlock since the SQL plugin is not available on IOS.
If any type can explain why SQL plug-ins are not supported in IOS, I would appreciate an explanation.

**Comment by FabianLars** [comment]
Explanation is simple, nobody on the team looked into it yet nor tested it afaik. I looked at the sqlx issues/discussions for iOS specific things but didn't find anything noteworthy.

Maybe it already works, maybe the sqlite implementation needs a change to the path for the database, maybe the xcode project needs some changes for additional libs (doubt it), or worst case maybe sqlx as a whole doesn't work on iOS.

**Comment by jfahrenkrug** [comment]
@FabianLars I just added the sql plugin to a sample app and ran it in the iOS simulator. It compiled and I'm able to open a DB and select stuff. Maybe the the docs can be updated to reflect that it works? Maybe some others and can try it to confirm.
